Sources/Data Model/Project.swift

```diff
@@ -42,6 +42,7 @@ struct Project: Codable, Equatable {
     var typedAudiences: [Audience]?
     var featureFlags: [FeatureFlag]
     var botFiltering: Bool?
+    var sendFlagDecisions: Bool?
```

Comment:
It's not nullable.

Reply:
Since `Project` struct is marked codable, for every value that might not exist, we have to mark it as optional, otherwise decoding a datafile with no `sendFlagDecisions` into `Project` struct will cause a crash.